lib/package/sqs/compileMethodsToSqs.js

```diff
@@ -47,31 +47,16 @@ module.exports = {
   },
 
   getSqsMethodIntegration(http) {
-    let queueName = http.queueName
-    if (typeof http.queueName == 'string') {
-      queueName = `"${queueName}"`
-    }
     const integration = {
       IntegrationHttpMethod: 'POST',
       Type: 'AWS',
       Credentials: {
         'Fn::GetAtt': ['ApigatewayToSqsRole', 'Arn']
       },
       Uri: {
-        'Fn::Join': [
-          '',
-          [
-            'arn:aws:apigateway:',
-            {
-              Ref: 'AWS::Region'
-            },
-            ':sqs:path//',
-            {
-              Ref: 'AWS::AccountId'
-            },
-            '/',
-            queueName
-          ]
+        'Fn::Sub': [
+          'arn:aws:apigateway:${AWS::Region}:sqs:path//${AWS::AccountId}/${queueName}',
```

Comment:
the old code puts a `"` around the string value - e.g. `"myQueue"` but the new code doesn't, is that the correct?

Reply:
same here.